### DISCUSSION TITLE
Upgrading from 1.0.0 to 1.6.0

### DISCUSSION THREAD

**Discussion by Ramemis** [OrchardCMS/OrchardCore#14055] [General]
We upgraded the orchard core package and the dotnet core from 3.1 to 6.0 the changes are only on the package reference (1.6.0 packages)and the target framework from 3.1 to 6.0. But on upgrading the package reference I started to see the setup page.  Not sure why I'm getting this screen I already have the db in place for this. Is there any changes need to be done as part of migration.

![image](https://github.com/OrchardCMS/OrchardCore/assets/121164498/a1687508-a2f8-4e14-b2bc-315f64f64510)

**Comment by hishamco** [comment]
I remember we did some changes in the database since `1.0.0`, @MikeAlhayek did most of them if I recall

**Comment by MikeAlhayek** [comment]
Not sure. It sounds like App_Data folder is empty or it does not see it for some reason. I don't know if the location of the appsettings changed since. Maybe review the release notice of each version and look for breaking changes

**Comment by MichaelPetrinolis** [comment]
I suggest on a running 1.0.0 instance to start subsequent upgrades.
If something goes wrong, focus on the specific version upgrade.

**Comment by Ramemis** [comment]
@MichaelPetrinolis  Thanks for your guidance. As you suggested I can successfully upgraded until 1.4.0 on upgrading to 1.5.0 I'm facing the above setup issue where the I can only see the setup page. I looked into the 1.5.0 breaking changes doc and tried changes like adding Trustedcertificate = true and also done the lucene sql update changes. But I still face the issue. Also I believe in local its working fine. When I deploy to app service via docker publish thats where I face this issue. Anything I missed.

**Reply by MikeAlhayek** [reply]
@Ramemis have you looked at your logs? If the connection is failing after the YesSql upgrade, you should see error in the log file.

**Comment by MichaelPetrinolis** [comment]
check #12903, it is TrustServerCertificate=true

**Comment by Ramemis** [comment]
@MichaelPetrinolis This is the connection string which has TrustServerCertificate = True. Even after appending it facing the issue. Anything need to be corrected here.
`Data Source=Ipaddress;Initial Catalog=db;User Id=userid;Password=pswd;Trusted_Connection=False;Encrypt=True;Connection Timeout=30;Enlist=False;MultipleActiveResultSets=True;ConnectRetryCount=5;ConnectRetryInterval=5;TrustServerCertificate=True`

**Reply by MikeAlhayek** [reply]
try changing `Encrypt=True;`  to `Encrypt=False`
